doc/schema/README.rst

```diff
@@ -90,5 +90,6 @@ kw - keyword element, name is given as an attribute. Type attribute describes th
     * ``msg`` - any number of optional keyword messages
     * ``timeout`` - optional keyword timeout. Before 3.0 this was an attribute.
     * ``status`` - keyword has to have a status
+    * ``sourcename`` - name of keyword with embedded arguments, from it's definition
```

Comment:
"it's" -> "its"

Could also be changed to something like

> original name of the keyword when using embedded arguments, not set otherwise. New in 4.0.

altogether.

Reply:
Done